src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectFactory.cs

```diff
@@ -35,7 +35,7 @@ private IProject GetMSBuildProj(string projectDirectory, NuGetFramework framewor
         {
             var msBuildExePath = _environment.GetEnvironmentVariable(Constants.MSBUILD_EXE_PATH);
```

Comment:
I thought it would be weird that it wouldn't match the constant name, which is why I kept it as is. I can't change the constant because that's what MSBuild has, by the way.